src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import './App.css';
-// import from 'react';
-import { ChakraProvider } from '@chakra-ui/react'
+import EmailSending from './components/EmailTemplates/EmailSending';
+import { ChakraProvider, Text } from '@chakra-ui/react'
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Text' is defined but never used.